test/e2e/specs/specs-playwright/wp-support__showme-where.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @group calypso-pr
+ */
+
+import {
+	DataHelper,
+	LoginFlow,
+	SidebarComponent,
+	SupportComponent,
+	setupHooks,
+	GutenboardingFlow,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( 'Support: Show me where' ), function () {
+	let page: Page;
+
+	setupHooks( ( args: { page: Page } ) => {
+		page = args.page;
+	} );
+
+	describe.each( [
+		{ siteType: 'Simple', user: 'defaultUser' },
+		{ siteType: 'Atomic', user: 'wooCommerceUser' },
+	] )( 'Search and view a support article ($siteType)', function ( { user } ) {
+		let supportComponent: SupportComponent;
+
+		it( 'Log in', async function () {
+			const loginFlow = new LoginFlow( page, user );
+			await loginFlow.logIn();
+		} );
+
+		it( 'Navigate to Tools > Marketing', async function () {
+			const sidebarComponent = new SidebarComponent( page );
+			await sidebarComponent.navigate( 'Tools', 'Marketing' );
+		} );
+
+		it( 'Open support popover', async function () {
+			supportComponent = new SupportComponent( page );
+			await supportComponent.openPopover();
+		} );
+
+		it( 'Displays default entries', async function () {
+			await supportComponent.defaultStateShown();
+		} );
+
+		it( 'Enter search keyword', async function () {
+			const keyword = 'domain';
+			await supportComponent.search( keyword );
+		} );
```

Comment:
I think these cases are already covered by the other support popover e2e tests, what do you think of taking them out of here to reduce duplication?